package.json

```diff
@@ -34,6 +34,19 @@
     "open formula",
     "openformula"
   ],
+  "browserslist": [
+    "maintained node versions",
+    "last 2 chrome versions",
+    "last 2 and_chr versions",
+    "last 2 firefox versions",
+    "last 2 and_ff versions",
+    "last 2 safari versions",
+    "last 2 ios_saf versions",
+    "last 2 edge versions",
+    "last 2 and_uc versions",
+    "last 2 and_qq versions",
+    "last 2 op_mob versions"
+  ],
```

Comment:
Based on the ['Supported Browsers' page in the docs](https://hyperformula.handsontable.com/guide/supported-browsers.html#list-of-supported-browsers).